python/pdstools/utils/cdh_utils.py

```diff
@@ -325,6 +349,14 @@ def getMatches(files_dir, target):
         match = [file for name in names if re.findall(name.casefold(), file.casefold())]
         if len(match) > 0:
             matches.append(match[0])
+    if len(matches) == 0:
+        raise FileNotFoundError(
+            """Couldn't find a file with default ADM table names. Please try feeding model and predictor files seperately. \n
```

Comment:
I'd just fstring in the given file path here, just so it's explicit _where_ we couldn't find the files.